### DISCUSSION TITLE
Polling interval change to 1 second

### DISCUSSION THREAD

**Discussion by vicky1631** [milo526/homebridge-tuya-web#506] [Ideas]
Hi, the minimum polling interval is set to 600 which voids the usage of automation on HomeKit app since for example if I turn on a light using voice then for the next 600 seconds on the HomeKit this particular light will be in OFF status. If the polling interval is set as 1 there would be instant update if the device status changes.. Also IR devices, locks and gateways added to smart life app are not recognised by home bridge.. Pls help me get this also on HomeKit app

**Comment by milo526** [comment]
This is not possible due to limitations with the API.

If we request data more than once every minute; all we will get is errors. That is why the minimum is hardcode at just over a minute.

The other devices are probably not supported if they don't show up but you can paste the api output here and I'll take a look at it.
